firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/ProviderProxyNativeComponentTest.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.crashlytics.internal;
+
+import static org.mockito.ArgumentMatchers.eq;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+public class ProviderProxyNative {
+  private static final String TEST_STRING = "abc";
+  private static final long TEST_LONG = 123;
+  private static final boolean TEST_BOOLEAN = true;
+  private static final int TEST_INT = 1234;
+
+  @Mock private CrashlyticsNativeComponent component;
+
+  @Before
+  public void setUp() throws Exception {
+    MockitoAnnotations.initMocks(this);
+  }
```

Comment:
nit: for this simple case you can just use the more concise:
```suggestion
  private final CrashlyticsNativeComponent component = mock(CrashlyticsNativeComponent.class);
```